cmd/fyne/internal/commands/package-wasm.go

```diff
@@ -71,7 +72,16 @@ func (w webData) packageWebInternal(appDir string, exeWasmSrc string, icon strin
 		return err
 	}
 
-	wasmExecSrc := filepath.Join(runtime.GOROOT(), "misc", "wasm", "wasm_exec.js")
+	goroot, err := GOROOT()
+	if err != nil {
+		return err
```

Comment:
This also could lead to the test failure as the copy could not previously exit at this point...